Neos.Ui/core/src/application/LinkTypes/PhoneNumber/PhoneNumber.tsx

```diff
@@ -99,6 +106,13 @@ export const PhoneNumber = makeLinkType<PhoneNumberLinkModel, PhoneNumberLinkOpt
                 <div style={{ display: 'grid', gridTemplateColumns: '160px 1fr', minWidth: '600px' }}>
                     <Field<string>
                         name='countryCallingCode'
+                        format={value => {
+                            (value !== undefined || value !== '') && setCodeArea(value)
```

Comment:
What should this code do? I think you forgot to delete this part.

Reply:
The format property fixes the deleting problem because when we click on the delete button the value will be undefined and this line helps us to set the previous value. BTW. I change the code for more readability.